browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -118,6 +136,11 @@ int OnBeforeURLRequest_StaticRedirectWork(
 
       URLPattern(URLPattern::SCHEME_HTTPS, "https://crlsets.brave.com/*"),
       URLPattern(URLPattern::SCHEME_HTTPS, "https://crxdownload.brave.com/*"),
+
+      // Brave's translation relay server
+      URLPattern(URLPattern::SCHEME_HTTP, "http://localhost:8195/*"),
```

Comment:
Might be nice to have constants for these.

Reply:
Addressed, by the way, the proxy server below is actually not needed, so removed.